CHANGELOG.md

```diff
@@ -2,10 +2,11 @@
 
 ## master
 
-## [v5.2.0][v5.2.0] (December 10, 2021)
+## [v5.2.0][v5.2.0] (December 13, 2021)
 
-* Refactored gin middleware([#224](https://github.com/airbrake/gobrake/pull/224))
-* Added the support for using severity levels defined by [Apex/log](https://github.com/apex/log) middleware instead of custom defined severity levels.([#225](https://github.com/airbrake/gobrake/pull/225))
+* Deprecating `NewMiddleware` func and replacing with `New` func in gin middleware.([#224](https://github.com/airbrake/gobrake/pull/224))
+
+* Use [Apex/log](https://github.com/apex/log) severity levels instead of custom defined severity levels.([#225](https://github.com/airbrake/gobrake/pull/225))
```

Comment:
```suggestion
* Used [Apex/log](https://github.com/apex/log) severity levels instead of custom defined severity levels ([#225](https://github.com/airbrake/gobrake/pull/225))
```